### DISCUSSION TITLE
Install Latest Version Of Podman CLI Inside Podman Desktop

### DISCUSSION THREAD

**Discussion by trymeouteh** [podman-desktop/podman-desktop#10464] [Ideas]
On some Linux distros such as Ubuntu and Linux Mint we do not have access to the latest version of Podman CLI. It would be awesome of there was a way to install the latest version of Podman CLI from the Podman Desktop settings to get the latest version of Podman CLI

**Comment by afbjorklund** [comment]
Podman (and Podman Desktop) does not _want_ to provide binaries for Linux, that is up to the distros - and to the user:

* https://github.com/podman-desktop/podman-desktop/discussions/5762

Even if the CLI/client was included (like it is on the other platforms), you would also need the server or a VM/machine*?

\* Currently the workaround is to run `podman machine` from the CLI as well, if it is needed...

---

There are multiple discussions about installing podman on ubuntu, in the other repo:

https://github.com/containers/podman/discussions?discussions_q=is%3Aopen+ubuntu

Some earlier discussion about bringing the package repository back (didn't happen):

* https://github.com/podman-desktop/podman-desktop/issues/5317

**Comment by afbjorklund** [comment]
It is documented how to access the Podman Machine from Ubuntu **on Windows**, but not on Linux:

https://podman-desktop.io/docs/podman/accessing-podman-from-another-wsl-instance

* https://github.com/podman-desktop/podman-desktop/issues/6225
